hazelcast/src/test/java/com/hazelcast/config/NativeMemoryConfigTest.java

```diff
@@ -1,41 +0,0 @@
-/*
- * Copyright (c) 2008-2022, Hazelcast, Inc. All Rights Reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package com.hazelcast.config;
-
-import com.hazelcast.test.HazelcastParallelClassRunner;
-import com.hazelcast.test.annotation.ParallelJVMTest;
-import com.hazelcast.test.annotation.QuickTest;
-import nl.jqno.equalsverifier.EqualsVerifier;
-import nl.jqno.equalsverifier.Warning;
-import org.junit.Test;
-import org.junit.experimental.categories.Category;
-import org.junit.runner.RunWith;
-
-import static com.hazelcast.test.HazelcastTestSupport.assumeDifferentHashCodes;
-
-@RunWith(HazelcastParallelClassRunner.class)
-@Category({QuickTest.class, ParallelJVMTest.class})
-public class NativeMemoryConfigTest {
-
-    @Test
-    public void testEqualsAndHashCode() {
-        assumeDifferentHashCodes();
-        EqualsVerifier.forClass(NativeMemoryConfig.class)
-                      .suppress(Warning.NULL_FIELDS, Warning.NONFINAL_FIELDS)
-                      .verify();
-    }
-}
```

Comment:
Why did you delete this?

Reply:
I don't really remember, probably by mistake... 316de23b436bfd2578cf7ef48ea185be80dcfe23